libraries/adsk/materials/wood/parquet_3/ND_parquet_03_material.mtlx

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<materialx version="1.37">
```

Comment:
The shaders in this definition are now discoverable when searching for shaders. Works in MaterialXView and will work in LDX when I merge this over.